.github/workflows/python.yml

```diff
@@ -86,16 +86,5 @@ jobs:
         cd src/main/python
         ls tests/
         echo "Beginning tests"
-        python tests/test_matrix_binary_op.py
-        echo "Exit Status: " $?
-        sleep 3
-        python tests/test_matrix_aggregations.py
-        echo "Exit Status: " $?
-        sleep 3
-        python tests/test_l2svm.py
-        echo "Exit Status: " $?
-        python tests/test_lineagetrace.py
-        sleep 3
-        echo "Exit Status: " $?
-        python tests/test_l2svm_lineage.py
+        python -m unittest tests/*.py
```

Comment:
for federated we have to adapt

Reply:
For the federated i have made a separate folder, since we need the workers to start up. This fits perfectly with that. Furthermore, i think in the future we need to separate the tests into sub-folders. But this is perfect for now.